### DISCUSSION TITLE
IPTV m3u playlist not loading

### DISCUSSION THREAD

**Discussion by Yamilcfg** [AndreyPavlenko/Fermata#196] [General]
Hello,

I have an IPTV m3u playlist and it won't load using the TV addon. When I add the m3u playlist I get 0 channels and 0 groups. What I'm doing wrong? any specific suggestions?? Thanks in advance

**Comment by AndreyPavlenko** [comment]
Can you share the playlist so I could check?

**Reply by Yamilcfg** [reply]
http://ky-iptv.com:25461/get.php?username=xxxxxxx&password=xxxxxxxx&output=ts&type=m3u

**Reply by AndreyPavlenko** [reply]
Could you download the playlist and send me by email?

**Reply by Yamilcfg** [reply]
I've been trying but is not downloading, also contacted the IPTV provider for support but no answer yet ...... any workaraound?

**Reply by AndreyPavlenko** [reply]
If you can't download it, then, obviously, Fermata can't do it either. Please, contact your provider.
As a workaround, try changing the user agent to Google.

**Reply by Yamilcfg** [reply]
I have the playlist file, to what email do you want me to send it?

**Reply by AndreyPavlenko** [reply]
First of all, try doing it yourself - just add the file instead of URL.

**Reply by Yamilcfg** [reply]
Did that, it loads but it seems the playlist is too big for Fermata to handle, the app just keeps crashing, any suggestion?

**Reply by AndreyPavlenko** [reply]
Please, send me by email andrey.a.pavlenko@gmail.com

**Reply by Yamilcfg** [reply]
Sent you an email with a link to dowanload the file

**Reply by Yamilcfg** [reply]
I haven't received anything from you ............ have you managed to do something with the playlist?

**Reply by AndreyPavlenko** [reply]
I haven't received anything.

**Comment by jhonherry1122** [comment]
I suggested to you the best IPTV services provider website. You may visit this website  https://www.arenaiptv.shop/ and tell them your issue I hope they help you.
